lib/insights/api/common/rbac/utilities.rb

```diff
@@ -3,15 +3,6 @@ module API
     module Common
       module RBAC
         module Utilities
-          def validate_groups
```

Comment:
@mkanoor You mentioned the other methods in the module are no longer used, correct?  Do we remove this entire file?

Reply:
I think even if we do, it probably doesn't belong in the scope of this PR (as small of a change as that is)

Reply:
@gmcculloug We can do that in a separate PR, we also need @hsong-rh to update the Approval code to use this new class instead of the old mixin and we can then have a PR to delete it for good.

Reply:
@eclarizio Can we not touch this file for now and remove it in a different PR, if we remove it now it will break Approval. Once this is merged we would have to wait for @hsong-rh and then get rid of this file entirely.

Reply:
Sure, that's fine.